src/ui/public/courier/data_source/_abstract.js

```diff
@@ -201,8 +203,15 @@ define(function (require) {
      *****/
 
     SourceAbstract.prototype._myQueued = function () {
```

Comment:
Is anything actually using `_myQueued` now? Is this something we can just ditch entirely?